pyiron_contrib/atomistics/fitsnap/job.py

```diff
@@ -98,4 +98,4 @@ def to_hdf(self, hdf=None, group_name=None):
         super(FitsnapJob, self).to_hdf(hdf=hdf, group_name=group_name)
 
     def from_hdf(self, hdf=None, group_name=None):
-        super(FitsnapJob, self).from_hdf(hdf=hdf, group_name=group_name)
+        super(FitsnapJob, self).from_hdf(hdf=hdf, group_name=group_name)
```

Comment:
I didn't see this in the original PR, but this shouldn't be necessary if you're just calling `super()` anyway.  Is there something weird going on, that made it necessary?  If so we should look closer to the source.